PCbuild/_testinternalcapi.vcxproj

```
@@ -108,6 +108,12 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions Condition="'$(UseJIT)' == 'true'">_Py_JIT;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(UseJIT)' == 'true'">_Py_TIER2;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

I *think* these can be combined:

```suggestion
      <PreprocessorDefinitions Condition="'$(UseJIT)' == 'true'">_Py_JIT;_Py_TIER2;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

---

Yes, they can, but I won't, in anticipation of changing the second one to `UseTIER2` to enable the Tier 2 interpreter on Windows. (I've got this half working. :-)